lib/AS3935MI/src/AS3935MI.h

```diff
@@ -175,11 +176,11 @@ class AS3935MI
 	void writePowerDown(bool enabled);
 
 	/*
-	@return true if disturbers are masked, false otherwise. */
+	@return true if disturbers are masked, false otherwisee. */
```

Comment:
S&R side-effect?

Reply:
Yep, should have clicked the match-whole-word box